### ISSUE TITLE
No notifications for email which skip inbox and are moved to folders

### ISSUE THREAD

**Issue by hazim1093** [nylas/N1#2356]
In gmail, I have set rules for some specific type of emails that they skip inbox and get routed to another folder.

e.g.
emails with sender containing "github" go in the Github folder and do not appear in my inbox.

I don't get popup notifications for such emails.
I'm using Ubuntu 16.04

**Comment by bengotow** [comment]
Hi @hazim1093 this is intended behavior, because many users direct mail to the archive, to spam, etc. so they don't receive notifications. Are there other mail clients that used to do this for you?

**Comment by hazim1093** [comment]
I'm not sure about the default behavior of thunderbird, but it sure has an awesome addon that allows to customize notifiactions/sound/actions when mail is received in different folders. 

I think it would be great to have something like that

[https://addons.mozilla.org/en-US/thunderbird/addon/mailbox-alert/](link to addon)

**Comment by mjuarez** [comment]
I'm having the inverse of this issue I believe. I have items skipping inbox into folders and still get notified.

OS X 10.11.6, Nylas 0.4.51-93ef779 (0.4.51-93ef779)

**Comment by bbbenji** [comment]
Yes, it would be nice to receive notifications for emails that skip the inbox and are directed to a folder.

Airmail for example allows for this.
